Update all documentation for 2.7.0 (#7533)
keepassxreboot/keepassxc: docs/topics/AutoType.adoc

=== Configure Global Auto-Type
You can define a global Auto-Type hotkey that starts the Auto-Type process. To configure the hotkey, perform the following steps:

Navigate to _Tools_ -> _Settings_ -> Auto-Type tab *(1)*. Click into the _Global Auto-Type shortcut_ box and press the desired key combination that will trigger the Auto-Type process *(2)*.

.Auto-Type settings
image::autotype_settings.png[]

You can configure additional Auto-Type settings in this window such as start delay, inter-key typing delay, and matching options. If Auto-Type is not working well for you, try adjusting the default delays.

> A new feature has been added to remember the last used entry for a short period of time between presses of the global Auto-Type hotkey. This means you can type different sequences or parts of sequences during a login workflow without having to find the specific entry again.

Reviewer: Shouldn't the documentation just document what the current state is instead of highlighting new features? Just asking :sweat_smile:

Author: Good point, this was one of my first changes and I stopped doing that elsewhere.

=== Configure Auto-Type Sequences
Each entry in your database can have multiple Auto-Type sequences associated with various window titles. Simulated key presses can be sent to any other currently open window of your choice (web browser windows, login dialogs boxes, and so on). When the Global Auto-Type hotkey is pressed, KeePassXC will search your database for entries matching the current selected window title.

NOTE: The default Auto-Type sequence is `{USERNAME}{TAB}{PASSWORD}{ENTER}`. This means that it first types the username of the selected entry, then presses the `Tab` key, then types the password of the entry and finally presses the `Enter` key.

> TIP: To change the default Auto-Type sequence for all entries of your database, edit the root (top-most) group of your database and set a specific sequence. This will be pushed down to all groups and entries that are inheriting settings from their parent.

Reviewer: The wording of the last sentence is a bit odd, not sure if it can be improved.

> TIP: To change the default Auto-Type sequence for all entries of your database, edit the root (top-most) group of your database and set a specific sequence. This will be pushed down to all groups and entries that are inheriting settings from their parent.

Reviewer 2: ```suggestion
TIP: To change the default Auto-Type sequence for all entries of your database, edit the top-most group of your database and set a specific sequence. Sequences defined on that group are inherited by all groups and entries below it.
```

To configure Auto-Type sequences for your entries, perform the following steps:

1.	Navigate to the entries list and open the desired entry for editing. Click the _Auto-Type_ item from the left-hand menu bar *(1)*. Press the `+` button *(2)* to add a new sequence entry. Select the desired window using the drop-down menu, or simply type a window title in the box *(3)*.
+

> TIP: You can use an asterisk (`\*`) to match any value (e.g., when a window title contains a dynamic filename or website name). Set the window title to `*` to match all windows. Leave the window title blank to offer additional default Auto-Type sequences, such as custom attributes.

Reviewer: Good note for blank titles without adding new tips :+1:

> TIP: You can use an asterisk (`\*`) to match any value (e.g., when a window title contains a dynamic filename or website name). Set the window title to `*` to match all windows. Leave the window title blank to offer additional default Auto-Type sequences, such as custom attributes.

Reviewer 2: Last sentence is unclear.

.Additional Auto-Type choices
image::autotype_selection_dialog_type_menu.png[,70%]

> The option to type just the username, password, or current TOTP value is available by right clicking the desired row or expanding the Type Sequence button options. You can also copy these values to the clipboard (not recommended).

Reviewer: If we say we don't recommend something we need to explain why somewhere.

> The option to type just the username, password, or current TOTP value is available by right clicking the desired row or expanding the Type Sequence button options. You can also copy these values to the clipboard (not recommended).

Reviewer 2: ```suggestion
The option to type just the username, password, or current TOTP value is available by right-clicking the desired row or expanding the Type Sequence button options. You can also copy these values to the clipboard (not recommended).

```

> TIP: On Windows, you will see an option to use a virtual keyboard in this sub-menu. This is an experimental feature that allows you to type into virtual machines by simulating actual keyboard presses. Some international keyboards may be unsupported due to limitations in the Windows API.

Reviewer: Well put :+1:
